### ISSUE TITLE
Amazon EKS optimized accelerated Amazon Linux AMIs - How are these created? Where are the build Scripts?

### ISSUE THREAD

**Issue by scottblack1** [awslabs/amazon-eks-ami#1339]
I have a requirement to pin Nvidia driver versions on the EKS 1.25 GPU AMI. This essentially is to stop the driver version changing under us as there are significant tests we must do to validate when moving to a new driver.

Because of this requirement, for the last few years and previous k8s version I have built a GPU AMI on top of the standard EKS AMI by installing the nvidia-drivers and nvidia-docker etc. With the move to k8s 1.25 looming, I am looking to do the same for the 1.25 GPU image but I have had difficulties determining what configuration changes etc are required for containerd.

This repo contains the build scripts for the AMIs and I see both GPU / Standard AMIs are in the release notes but I cannot understand how the nvidia components etc are installed. Is there another build script that has this code that could be shared please?

TLDR; Where is nvidia / gpu specific configuration? How are these installed in the AMI? 

Thanks,

Scott

**Comment by cartermckinnon** [comment]
The GPU AMI template is not currently open source. We're tracking this request in #674

**Comment by cartermckinnon** [comment]
We *are* planning to update the NVIDIA drivers in the official builds: https://github.com/awslabs/amazon-eks-ami/issues/1060#issuecomment-1625391327
